README.md

```diff
@@ -85,6 +85,12 @@ The region the AWS `bucket` is located in.
 
 *Default:* `us-east-1`
 
+### acl
+
+The ACL to apply to the objects.
+
+*Default:* `public-read`
```

Comment:
You are right. It's not really explicit. The default already exists in https://github.com/ember-cli-deploy/ember-cli-deploy-s3/blob/59e083a73141bf17e67a83981ded573ff8da9caf/lib/s3.js#L68

But I would prefer to move the default to another place too. I will look into the other defaults.